target/linux/lantiq/patches-5.4/0110-MIPS-lantiq-dma-add-small-deley-after-reset.patch

```diff
@@ -0,0 +1,32 @@
+From 4f4fa80cd39e7b29d9364183d564f754f76b86e6 Mon Sep 17 00:00:00 2001
+From: Aleksander Jan Bajkowski <olek2@wp.pl>
+Date: Sat, 19 Jun 2021 13:38:12 +0200
+Subject: [PATCH 1/5] MIPS: lantiq: dma: add small deley after reset
```

Comment:
typo "deley" in patch name/subject; same for 5.10

Reply:
Thanks. This has been fixed.